osu.Game/Beatmaps/Drawables/BeatmapPanel.cs

```diff
@@ -65,15 +69,11 @@ public BeatmapPanel(BeatmapInfo beatmap)
                 {
                     RelativeSizeAxes = Axes.Both,
                 },
-                new Triangles
+                triangles = new Triangles
                 {
-                    // The border is drawn in the shader of the children. Being additive, triangles would over-emphasize
-                    // the border wherever they cross it, and thus they get their own masking container without a border.
-                    Masking = true,
-                    CornerRadius = Content.CornerRadius,
                     RelativeSizeAxes = Axes.Both,
-                    BlendingMode = BlendingMode.Additive,
-                    Colour = new Color4(20, 43, 51, 255),
+                    ColourLight = OsuColour.FromHex(@"3a7285"),

```

Comment:
yeah, quite possibly. i haven't yet figured out if flyte's triangle ranges are consistent across elements though (i tend to think they aren't atm).